src/internal/clusterstate/current.go

```diff
@@ -51,4 +52,11 @@ var DesiredClusterState migrations.State = migrations.InitialState().
 	}).
 	Apply("create identity config table v0", func(ctx context.Context, env migrations.Env) error {
 		return identity.CreateConfigTable(ctx, env.Tx)
+	}).
+	Apply("create auth schema", func(ctx context.Context, env migrations.Env) error {
+		_, err := env.Tx.ExecContext(ctx, `CREATE SCHEMA auth`)
+		return err
+	}).
+	Apply("auth tokens", func(ctx context.Context, env migrations.Env) error {
```

Comment:
I think the convention we're following for the migration names is like `create auth tokens table v0`